document stage:add (#2208)
iterative/dvc.org: content/docs/command-reference/stage/index.md

> # plots

Reviewer: title?

> A set of commands to add and list stages:

Reviewer: use tooltip here `<abbr>` for the stages term?

> A set of commands to add and list <abbr>stages</abbr>:
> [add](/doc/command-reference/stage/add).

Reviewer 2: A bit more like https://dvc.org/doc/command-reference/cache since there's just 1 subcommand for now:
```suggestion
Contains a helper command to set the create or update <abbr>stages</abbr>:
[add](/doc/command-reference/stage/add).
```

Reviewer 2: Wait. You mention `stage list` in the Description so I guess just put that in this intro instead of my suggestion? Or use my suggestion for now and remove the mention of `stage list` later (I assume there will be another PR for that anyway).

Author: Let's mention `stage list`, I'll try creating a PR today.

## Synopsis

```usage

> usage: dvc stage [-h] [-q | -v] {add,list}  ...

Author: If I remove the "extra spacing" here, syntax highlighting fails.

```suggestion
usage: dvc stage [-h] [-q | -v] {add,list} ...
```

If I add a spacing anywhere, for example as following, the syntax highlighting works, but not if I remove it.
```suggestion
usage: dvc stage [-h] [-q | -v] {add, list} ...
```

Reviewer 2: Hmmm strange but you can leave this line exactly as printed by `-h` 👍

positional arguments:
  COMMAND
    add         Create stage.
    list        List stages.
```

## Description

_Stages_ represent individual data processes, including their input and
resulting outputs. They can be combined to capture simple data workflows,
organize data science projects, or build detailed machine learning pipelines.

> `dvc stage add` can be used to create/update stages in the `dvc.yaml` file.
> Similarly, `dvc stage list` helps listing the stages present in the `dvc.yaml`
> file.

Reviewer 2: ```suggestion
`dvc stage add` can be used to create/update stages in `dvc.yaml`.
```
